doc/BACKUP.md

```diff
@@ -418,13 +418,16 @@ This can be difficult to create remote replicas due to the latency.
 [pqsyncreplication]: https://www.postgresql.org/docs/13/warm-standby.html#SYNCHRONOUS-REPLICATION
 
 ## SQLite Litestream Replication
-`/!\` WHO SHOULD DO THIS: Casual users
 
-`/!\` **CAUTION** `/!\` This technique will only be safe on 0.10.2
-or later.
-Earlier versions will crash regularly with "database is locked" errors,
-as Litestream puts a read-shared lock on the database.
-0.10.2 adds a 60-second timeout for locking.
+`/!\` **CAUTION** `/!\` Previous versions of this document recommended
+this technique, but we no longer do so.
+According to [issue 4857][], even with a 60-second timeout that we added
```

Comment:
```suggestion
According to [issue 4857](https://github.com/ElementsProject/lightning/issues/4857), even with a 60-second timeout that we added
```

Reply:
This is valid Markdown syntax, where you put the text inside `[` `]` in a separate line, like so:

> ```markdown
> This is [google][].
>
> [google]: https://www.google.com/
> ```

translates to:

> This is [google][].
>
> [google]: https://www.google.com/

Reply:
Oh my bad! Thanks to fix me on that